toolchain/valgrind.py

```diff
@@ -48,7 +48,7 @@ def setUp(self):
             if not smm.check_installed(package) and not smm.install(package):
                 self.cancel('%s is needed for the test to be run' % package)
         tarball = self.fetch_asset(
-            "http://valgrind.org/downloads/valgrind-3.12.0.tar.bz2")
+                "ftp://sourceware.org/pub/valgrind/valgrind-3.13.0.tar.bz2")
```

Comment:
It would be nice if we had a lib to probe the latest version of a given asset in a given location so we could call `fetch_asset()` always with the latest version.

Reply:
@apahim seems me good idea  , i will send a patch for it

Reply:
@PraveenPenguin, the new `avocado.utils.vmimage` library has something like that to get the latest distro and image versions. Feel free to use it as reference ;)

Reply:
@apahim sure , thanks